common/src/main/java/net/adoptopenjdk/icedteaweb/i18n/SupportedLanguages.java

```diff
@@ -0,0 +1,16 @@
+package net.adoptopenjdk.icedteaweb.i18n;
+
+import java.util.Locale;
+
+public enum SupportedLanguages {
+    en("en"), cs("cs"), de("de"), pl("pl");
+    private Locale locale;
+
+    SupportedLanguages(String lang) {
+        this.locale = new Locale(lang);
```

Comment:
Maybe we check that locale is != null and throw a IllegalStateException otherwise.

Reply:
done